session4/src/com/itschool/homework/Concat.java

```diff
@@ -0,0 +1,24 @@
+package com.itschool.homework;
+
+public class Concat {
+        public static void main(String[] args) {
+
+            String text1 = "Ana are mere.";
+            //              0123456789123
+            String text2 = "Si Ionut are mere.";
+            String replace = text1.replace ("."," ");
+
+            System.out.println(replace + text2.toLowerCase());
+
+            String replace2 = text2.replace ("ionut", "Ionut");
```

Comment:
you tried something here, but this variable `replace2` is not used, therefore can be deleted.